app/_hub/kong-inc/json-threat-protection/overview/_index.md

```diff
@@ -0,0 +1,238 @@
+  --data "config.max_object_entry_name_length=7" \
+  --data "config.max_array_element_count=2" \
+  --data "config.max_string_value_length=6" \
+  --data "config.enforce_mode=block" \
+  --data "config.error_status_code=400" \
+  --data "config.error_message=BadRequest1"
+```
+
+The configuration fields have the following meanings:
+
+- `config.max_body_size=1024`: The request body must not exceed 1024 bytes.
+- `config.max_container_depth=2`: The maximum depth of the container is 2.
+- `config.max_object_entry_count=4`: The number of object entries must not exceed 4.
+- `config.max_object_entry_name_length=7`: The key for an object entry must not exceed 7 bytes.
+- `config.max_array_element_count=2`: The number of array elements must not exceed 2.
+- `config.max_string_value_length=6`: The length of string values must not exceed 6 bytes.
+- `config.enforce_mode=block`: Enables `block` mode, where the request will not be proxied to the upstream service if the JSON violates the above limits.
+- `config.error_status_code=400`: When a JSON violation occurs, Kong returns an error code `400`.
+- `config.error_message=BadRequest1`: When a JSON violation occurs, Kong returns the error message `BadRequest1`.
+
+**Tap Mode**
+In tap mode, the plugin will inspect the JSON in the request body, but if there are any violations of the limits, it will not block the request. Instead, it will log a warning and proxy the request to the upstream service. In other words, in tap mode, the plugin only monitors the traffic.
+
+If we want to enable Tap mode, we only need to set `config.enforce_mode` to `log_only`, like this:
+
+```
+config.enforce_mode=log_only
+```
+
+
+After sending the above request, we will receive the following response:
+
+```json
+{
+  "updated_at": 1718876232,
+  "created_at": 1718876232,
+  "consumer": null,
+  "config": {
+    "max_body_size": 1024,
+    "max_container_depth": 2,
+    "max_array_element_count": 2,
+    "max_object_entry_count": 4,
+    "max_object_entry_name_length": 7,
+    "max_string_value_length": 6,
+    "enforce_mode": "block",
+    "error_status_code": 400,
+    "error_message": "BadRequest1"
+  },
+  "protocols": [
+    "grpc",
+    "grpcs",
+    "http",
+    "https"
+  ],
+  "service": {
+    "id": "1c5ba231-a3fe-4884-b422-ce6395fa227f"
+  },
+  "name": "json-threat-protection",
+  "id": "d8ef3743-f97e-4c98-87f1-53621cb28958",
+  "tags": null,
+  "route": null,
+  "ordering": null,
+  "consumer_group": null,
+  "instance_name": null,
+  "enabled": true
+}
+```
+
+
+
+## Examples:
+
+This request is successfully proxied to returns a response back
+
+```bash
+curl -XPOST http://127.1:8000/ \
+  -H "Content-Type: application/json" \
+  -d '{"name": "Jason","age": 20,"gender": "male","parents": ["Joseph", "Viva"]}' 
+```
+
+The response is:
+
+```html
+200 OK
+```
+
+
+
+This request will be intercepted by the JSON-Threat-Protection plugin and will return the configured message.
```

Comment:
```suggestion

The following request is intercepted by the JSON Threat Protection plugin and returns the configured message instead of the standard response:
```